src/less/components.less

```diff
@@ -29,6 +29,6 @@
 @import 'components/dropdown-select/index';
 @import 'components/file/index';
 @import 'components/form/index';
-@import 'components/radio/index';
+@import 'components/search/index';
```

Comment:
Revert addition of `components/search/index`.  This file should only remove `components/radio/index`.

Reply:
Netlify is failing to build because `components/search/index` doesn't exist (migrated in another PR).

Reply:
Changes have been done, please review.